cloudapi/config.go

```diff
@@ -196,6 +196,14 @@ func NewConfig() Config {
 		AggregationOutlierIqrCoefLower: null.NewFloat(1.5, false),
 		AggregationOutlierIqrCoefUpper: null.NewFloat(1.3, false),
 	}
+
+	// v2 enables aggregation by default
+	if c.APIVersion.Int64 == 2 {
```

Comment:
A non-blocking suggestion is doing the opposite way. 

As we know, the v2 is the current default, so `AggregationWaitPeriod` and `AggregationWaitPeriod`  could be aligned with the v2 (enabled aggregation) in the initializing of config above, but the previous default values could be set if the APIVersion is 1 (error-prone) in a similar to that logic block.

Later when we'd like to remove the v1, it's easier and clearer that only the logical block `if c.APIVersion.Int64 == 1` could be removed.

Reply:
Yeah, I liked the suggestion. I submitted a change to address it.